README.md

```diff
@@ -209,6 +209,10 @@ To update your project do: `git pull` in the project folder
 
 ## How to run with Docker
 
+1. Build the image docker `build -t PokemonGo-Bot:latest .`
+2. Start a new container from this newly created image `docker run -d --name pgobot PokemonGo-Bot:latest`
+3. Check the logs in real-time `docker logs -f pgobot`
+
```

Comment:
No need to add the tag 'latest', as it's implicit when no tag is provided. Btw we should add a container's restart policy.